src/engine/BMInterface.php

```diff
@@ -338,6 +344,111 @@ public function count_pending_games($playerId) {
+                       v.button_id,
+                       v.button_name,
+                       v.n_rounds_won,
+                       v.n_rounds_drawn,
+                       v.n_rounds_won >= v.n_target_wins AS is_winner
+                FROM game_player_view AS v
+                LEFT JOIN game AS g
+                ON g.id = v.game_id
+                LEFT JOIN game_status AS s
+                ON g.status_id = s.id
+                WHERE v.game_id = :game_id
+                ORDER BY g.id
+NOWDOC;
+
+            $parameters = array(':game_id' => $gameId);
+            $columnReturnTypes = array(
+                'game_id' => 'int',
+                'status_id' => 'int',
+                'status' => 'str',
+                'n_target_wins' => 'int',
+                'position' => 'int',
+                'player_id' => 'int',
+                'player_name' => 'str',
+                'button_id' => 'int',
+                'button_name' => 'str',
+                'n_rounds_won' => 'int',
+                'n_rounds_drawn' => 'int',
+                'is_winner' => 'bool',
+            );
+            $rows = self::$db->select_rows($query, $parameters, $columnReturnTypes);
+
+            return self::read_game_summary_from_db_results($rows);
+        } catch (Exception $e) {
+            error_log(
+                'Caught exception in BMInterface::load_game_summary: ' .
+                $e->getMessage()
+            );
+            $this->set_message("Internal error while loading game summary.");
+            return NULL;
+        }
+    }
+
+    /**
+     * Read game summary from DB results
+     *
+     * @param int $playerId
+     * @param array $rows
+     * @return array
+     */
+    protected function read_game_summary_from_db_results($rows) {
+        $button_0 = '';
+        $name_0 = '';
+        $nwins_0 = 0;
+        $button_1 = '';
+        $name_1 = '';
+        $nwins_1 = 0;
+        $winner = '';
+
+        foreach ($rows as $row) {
+            if (0 === $row['position']) {
+                $button_0 = $row['button_name'];
+                $name_0 = $row['player_name'];
+                $nwins_0 = $row['n_rounds_won'];
+            } else {
+                $button_1 = $row['button_name'];
+                $name_1 = $row['player_name'];
+                $nwins_1 = $row['n_rounds_won'];
+            }
+
+            if ($row['is_winner']) {
+                $winner = $row['player_name'];
+            }
+        }
+
+        return array(
+            'gameId' => $rows[0]['game_id'],
+            'statusId' => $rows[0]['status_id'],
+            'status' => $rows[0]['status'],
+            'n_target_wins' => $rows[0]['n_target_wins'],
+            'button_0' => $button_0,
+            'name_0' => $name_0,
+            'nwins_0' => $nwins_0,
+            'button_1' => $button_1,
+            'name_1' => $name_1,
+            'nwins_1' => $nwins_1,
+            'ndraws' => $rows[0]['n_rounds_drawn'],
```

Comment:
This seems like a very different API structure from how we report wins and losses in a non-tournament game.  Compare to:

```
        $expData['playerDataArray'][0]['gameScoreArray']['W'] = 1;
        $expData['playerDataArray'][1]['gameScoreArray']['L'] = 1;
```

I know you're not building out the whole structure of information about the game, but historically you've been pretty careful to use an API data structure which treats the players as parallel, with a sub-array.  I think that's the right thing to do in this case too.